### ISSUE TITLE
Allocated % in the allocation of emissions page shows 100% allocated when not exactly 100%

### ISSUE THREAD

**Issue by dleard** [bcgov/cas-reporting#465]
#### Describe the Bug:
The % allocated for emissions to products shows 100% allocated when not _exactly_ 100%. This blocks submission of the page & can be confusing to users because the error shown says not 100% have been allocated, but the page shows 100% for all emissions. This happens because we allow 4 decimal places for the emissions allocated, but the % rounds that up when displaying 99.9999% allocated, but still blocks because it's not exactly 100%.

Consider:

emissions to be allocated: 1000
product 1 allocated: 499.9999
product 2 allocated: 500.0000
Allocated percent will show: 100%
Page submission will be blocked & error message will say not 100% of emissions have been allocated.

Potential solution:
Always round the Allocated % down
Increase the precision of the Allocated % display (may need too many decimal places to be a valid solution)

#### Impact: 3

<img src="https://www.getzephyr.com/sites/default/files/riksk%20matrix.jpg" alt="Bug Matrix" width="250" height="250"/>

#### Steps to reproduce the behaviour:

1. Go to allocation of emissions page & allocate emissions with 1/10000th off of 100%
2. See that allocated % shows 100 but the page submission is blocked

-

**Comment by dleard** [comment]
A thought: is % even what we want to display here? Or would an unallocated total make more sense and be more accurate?

**Comment by dleard** [comment]
@acatchpole @patriciarussellCAS Duplicate of #468 ?

**Comment by acatchpole** [comment]
yes. this issue is addressed in https://github.com/bcgov/cas-registration/pull/2637

**Comment by rdromey** [comment]
Confirmed addressed already
